azure/deploy-to-azure/azure-integration/azuredeploy-managementgroups.json

```diff
@@ -168,6 +168,9 @@
                     },
                     "newguid": {
                         "value": "[parameters('newguid')]"
+                    },
```

Comment:
We see other entities like child management groups referring to the management group by its ID, not its name. I think we should follow that pattern.